modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,115 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-aws_{context}"]
+
+= Adding an ARM64 compute machine set to your cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`). 
+* You created a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary.
```

Comment:
Should we mention here that the single architecture cluster here is an adm64 cluster? Just because we allow arm clusters to be created as well, it would be good to call that out.

Reply:
Yeah I agree! I think that detail would be good to add